CharlieBackend.Api/Controllers/AttachmentsController.cs

```diff
@@ -61,8 +68,11 @@ public async Task<IActionResult> GetAttachments([FromBody]AttachmentRequestDto r
         }
 
         /// <summary>
-        /// GET: api/attachments/{id}
+        /// Downloads attachment
```

Comment:
"Get attachment by Id" would be better description

Reply:
Maybe even "Downloads attachment by Id" will be more descriptive I suppose

Reply:
Updated like this:
`Gets attachment by id`